Makefile

```diff
@@ -262,7 +251,6 @@ lint: $(GOLANGCI_LINT) ## Lint codebase
 modules: ## Runs go mod to ensure proper vendoring.
 	go mod tidy
 	cd $(TOOLS_DIR); go mod tidy
-	cd $(TEST_E2E_DIR); go mod tidy
```

Comment:
Do we not want/need a separate go module for e2e?

Reply:
Not anymore. We run kubetest in Docker instead of importing sonobuoy packages, which we also couldn't do because we can't simultaneously import the clusterctl framework from CAPI.